R/01-qc.R

```diff
@@ -1,39 +1,119 @@
 
-#' This is a title for a function
+#' A function to run QC
 #' @description This is a function here's where we describe what it does
-#' @param parameter Here's a parameter let's describe it here
+#' @param plasmid_cutoff default it NULL; `qc_plasmid_histogram` will calculate and set a default one unless a value is specified here 
```

Comment:
I think a more intuitive design for this argument would be to have it by default show the cutoff value in the argument and then if someone doesn't want a cutoff allow them to set it to NULL. This would add some clarity to what's happening with the cutoff. 

So for example, if the cutoff is based on a quartile have the default be the quartile and describe it that way. so the default is .25 or whatever. Then if someone doesn't want filtering they can set to NULL.